nodexml/README.md

```diff
@@ -0,0 +1,79 @@
+Node XML Schema
+===============
+
+
+Work-In-Progress
+----------------
+
+This readme, the schema, and the example document/s are all still a
```

Comment:
We could take this out, since once this is merged, we won't be planning major changes.